kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -39,8 +37,23 @@ window.Exercises = {
     RelatedVideos: {
         render: function() {}
     },
-    incompleteStack: [0]
-};
+    incompleteStack: [0],
+    proxyTrigger: function () {
```

Comment:
Oh, nevermind about upstream -- this is our file, yeah. But comment would be good.